datalad/distribution/get.py

```diff
@@ -237,6 +237,16 @@ def _get_flexible_source_candidates_for_submodule(ds, sm):
                         ds_repo.config[c])
                        for c in ds_repo.config.keys()
                        if c.startswith(candcfg_prefix)]:
+        # ensure that there is only one template of the same name
+        if type(tmpl) == tuple and len(tmpl) > 1:
+            lgr.debug('Found %i URL templates for the submodule clone '
```

Comment:
There's no need for that log message. We are trying to get rid of the various forms of double-reporting ;-)
If you throw an exception, then it either bubbles up, leading to its message being logged at ERROR level or it's catched and dealt with upstairs, in which case a proper logging of what's going on should happen at that spot (where we know the relevance of it).

Reply:
ah yeah, makes sense!

Reply:
removed it